packages/maskbook/src/components/InjectedComponents/SetupGuide.tsx

```diff
@@ -37,18 +36,21 @@ export enum SetupGuideStep {
     FindUsername = 'find-username',
     SayHelloWorld = 'say-hello-world',
 }
+
+const userGuideCompleted = localStorage.getItem('userGuideCompleted')
```

Comment:
@guanbinrui you're storing it in the SNS's local storage. don't do it.

Reply:
@Jack-Works #4536